src/silicone/utils.py

```diff
@@ -181,12 +181,21 @@ def _remove_t0_from_wide_db(times_needed, _db):
             _db.loc[model, scenario, time] = _db.loc[model, scenario, time] - offset
 
 
-def _make_interpolator(variable_follower, variable_leader, wide_db, time_col):
+def _make_interpolator(
```

Comment:
should this be in `utils` or just a private function in `interpolation`?

Reply:
it's a useful function on its own, I use it in a few notebooks. Arguably it shouldn't begin with the _ in this case

Reply:
Ok nice. If it's useful then making it public is also good (particularly given you've already got the tests, so maybe all it needs is documentation..)

Reply:
Ok nice. If it's useful then making it public is also good (particularly given you've already got the tests, so maybe all it needs is documentation..)

Reply:
I've added some documentation for it, I haven't renamed it as that would break the existing uses.